packages/evaluation/src/mcpclient/mobileWebMcpClient.ts

```diff
@@ -53,4 +69,58 @@ export class MobileWebMcpClient {
   async callTool(toolName: string, params: Record<string, unknown>) {
     return this.client.callTool({ name: toolName, arguments: params });
   }
+
+  private async initializeMobileWebMcpClient(): Promise<void> {
```

Comment:
This method should be renamed to something like `initializeMCPServer()`.

Reply:
ya, this method is not necessary, combined into connect method.